sys/compat/linprocfs/linprocfs.c

```diff
@@ -2092,6 +2092,97 @@ linprocfs_domax_map_cnt(PFS_FILL_ARGS)
 	return (0);
 }
 
+/*
+ * Filler function for proc/sys/fs/mqueue/msg_default
+ */
+static int
+linprocfs_domqueue_msg_default(PFS_FILL_ARGS)
+{
+	int res, error;
+	size_t size = sizeof(res);
+
+	error = kernel_sysctlbyname(curthread, "kern.mqueue.default_maxmsg",
```

Comment:
Why going through the sysctl instead of just exporting the variables and directly using their values in place?

Reply:
> Why going through the sysctl instead of just exporting the variables and directly using their values in place?

The problem is that they have generic names like maxmsg.  I can change them to have a `mqueue_` prefix.  Also, I've seen this practice of calling sysctl in other places.